hopsworks-common/src/main/java/io/hops/hopsworks/common/dao/jupyter/config/JupyterConfigFilesGenerator.java

```diff
@@ -169,23 +175,29 @@ public String pythonKernelPath(String kernelsDir, String pythonKernelName) {
     return kernelsDir + File.separator + pythonKernelName;
   }
   
-  public String createJupyterKernelConfig(Project project, JupyterSettings js, String hdfsUser)
+  public void createJupyterKernelConfig(Writer out, Project project, JupyterSettings js, String hdfsUser)
```

Comment:
This will break the enterprise edition. 
On K8s the templates are passed to the containers using a config map and never written to disk. Hence why all these methods return a string.

Reply:
I know, in CE the Writer is a FileWriter but in EE it's a StringWriter. I've got changes in EE too